### ISSUE TITLE
Keyboard navigation is impossible

### ISSUE THREAD

**Issue by ghidasy** [kimchi-project/kimchi#870]
Its impossible to navigate on Kimchi using only the keyboard. 

Naively trying to use Tabs is frustrating as the focus don't move to all elements, and once they exit the tabs bar its impossible to return with it. In the Guests screen it can only select the power-off button when the machine is off, and when the machine is on only the restart button from each machine, and in the Templates and Network screen it cannot move anywhere except to the create template button.

Using a keyboard helper (like vimperator) improves the situation, but does not solve it. All Drop-down menus are inaccessible without a mouse, even using the helpers, (except the ones in the "Networks" tab, that do work well). But if the user clicks in a drop-down, with a mouse, the open menu is accessible both by the keyboard helper and by tabs

The user menu is also inaccessible for the keyboard.

Its impossible to click in the VM window to open the NoVNC screen

**Comment by alinefm** [comment]
Hi @ghidasy 

Could you separate issues for the keyboard navigation?

Example, problems in the navigation bar should be reported on Wok as it is build by Wok.
Problems in Kimchi main content should be reported here.
Problems in Ginger main content should be reported on Ginger.
Problems in GingerBase main content should be reported on GingerBase.

In addition to the main content, we also need to make sure the keyboard navigation is allowed on input dialogs. When creating a guest, editing a guest, etc.

It is better to have all the issues properly described so we have a way to measure the amount of work we need to do to have Wok and its plugins complaint with accessibility guidelines.

**Comment by ghidasy** [comment]
Ok, will close this issue and create more specific ones
